compiler/src/dotty/tools/dotc/core/tasty/TreePickler.scala

```diff
@@ -665,6 +665,26 @@ class TreePickler(pickler: TastyPickler) {
               pickleTree(hi)
               pickleTree(alias)
           }
+        case tree @ Quote(body) =>
+          // Add QUOTE tag to TASTy
```

Comment:
I assume this should be "TODO: " ?
```suggestion
          // TODO: Add QUOTE tag to TASTy
```